client/app/scripts/vendor/term.js

```diff
@@ -2756,6 +2756,13 @@ Terminal.prototype.keyDown = function(ev) {
     case 123:
       key = '\x1b[24~';
       break;
+    // Special case for US-intl '/"
+    case 222:
+      key = '\'';
+      if (ev.shiftKey) {
+        key = '"';
+      }
+      break;
```

Comment:
I don't know how you are handling vendoring but, How do you make sure changes made directly are not lost when updating the library?